### PR TITLE
Work on Node.js workshop

### PR DESCRIPTION
**Pull request by Michael-Antczak**
**Note:** If you are adding a new lesson or have edited the name of a lesson file. Please make sure you update `./SUMMARY.md` to reflect the changes in the gitbook contents.

### CODE REVIEW

File: node-db/lesson1.md [hunk 1]
```diff
@@ -581,3 +615,9 @@ Take a look at the following links to learn more about Node.js.
 More about HTTP:
 [Tutsplus tutorial](https://code.tutsplus.com/tutorials/http-headers-for-dummies--net-8039)
 
+Learn more about JSON:
+* http://www.json.org
```
Comment (kabaros) [inline node-db/lesson1.md:619]: that json.org resource is very confusing

File: node-db/lesson1.md [hunk 2]
```diff
@@ -135,6 +135,42 @@ At the endo of the wizard, you should see a new file called `package.json` in yo
 
 Here is an example `package.json` file for a project called [Passport](https://github.com/jaredhanson/passport/blob/master/package.json).
 
+# What is JSON?
```
Comment (kabaros) [inline node-db/lesson1.md:138]: this is a better explanation but I think it doesn't need to come that early .. the location where it was fits better imo

Comment (Michael-Antczak) [reply node-db/lesson1.md:138]: I think the topic should be more or less explained at a point when it is introduced, and this is when we talk about package.json

File: node-db/lesson1.md (hunk 1 above)
Comment (Michael-Antczak) [reply node-db/lesson1.md:619]: Ok, so remove it then.

File: node-db/lesson1.md (hunk 2 above)
Comment (kabaros) [reply node-db/lesson1.md:138]: ok

**Review by kabaros — APPROVED** [review]

**Pull request merged**